### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by maskaravivek**
Fixes a few lint issues for issue #171

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java [hunk 1]
```diff
@@ -220,7 +221,7 @@ protected void onResume() {
     private void performLogin() {
         String username = usernameEdit.getText().toString();
         // Because Mediawiki is upercase-first-char-then-case-sensitive :)
-        String canonicalUsername = username.substring(0,1).toUpperCase() + username.substring(1);
+        String canonicalUsername = username.substring(0,1).toUpperCase(Locale.getDefault()) + username.substring(1);
```
Comment (whym) [inline app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java]: Perhaps you can call Utils.capitalize instead?

File: app/src/main/java/fr/free/nrw/commons/Utils.java [hunk 2]
```diff
@@ -303,11 +304,11 @@ public static String fixExtension(String title, String extension) {
         Pattern jpegPattern = Pattern.compile("\\.jpeg$", Pattern.CASE_INSENSITIVE);
 
         // People are used to ".jpg" more than ".jpeg" which the system gives us.
-        if (extension != null && extension.toLowerCase().equals("jpeg")) {
+        if (extension != null && extension.toLowerCase(Locale.US).equals("jpeg")) {
             extension = "jpg";
         }
         title = jpegPattern.matcher(title).replaceFirst(".jpg");
-        if (extension != null && !title.toLowerCase().endsWith("." + extension.toLowerCase())) {
+        if (extension != null && !title.toLowerCase(Locale.US).endsWith("." + extension.toLowerCase(Locale.US))) {
```
Comment (whym) [inline app/src/main/java/fr/free/nrw/commons/Utils.java]: title can be non-English (US English or anything else) on Wikimedia Commons, so it seems to be better to stick with the default locale.

extension here should be ASCII only, so Locale.US seems to make sense. (But did you also consider using Locale.ROOT?)

**Comment by domdomegg** [comment]
Just thought it might be more sensible to use `Locale.ENGLISH` instead of `Locale.US` - don't know if there  is any real difference but seeing as its the same as default Commons could be an idea.

File: app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java (hunk 1 above)
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java]: Right, done.

File: app/src/main/java/fr/free/nrw/commons/Utils.java (hunk 2 above)
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/Utils.java]: Made changes to use default locale as suggested.

**Comment by maskaravivek** [comment]
Thanks for the review @whym and @domdomegg. Have incorporate suggested changes. 

@domdomegg Yes, Using `Locale.ENGLISH` makes more sense. Have made this change.

**Comment by domdomegg** [comment]
Thank you @maskaravivek! :)

**Pull request merged**